main.go

```diff
@@ -18,11 +19,18 @@ type Claims struct {
 	Email string `json:"email"`
 }
 
+func getEnv(key, fallback string) string {
+	if value, ok := os.LookupEnv(key); ok {
+		return value
+	}
+	return fallback
+}
+
 var (
 	// default flag values
-	authDomain = ""
-	address    = ""
-	port       = 80
+	authDomain = getEnv("AUTH_DOMAIN", "")
+	address    = getEnv("LISTEN_ADDRESS", "")
+	port, err  = strconv.Atoi(getEnv("LISTEN_PORT", "80"))
```

Comment:
If we're not going to check it, let's replace `err` with `_` so we don't risk confusion with a pre-defined, top-level error variable